### PR TITLE
RES-1641 unused API calls

### PR DESCRIPTION
**Pull request by edwh**
Removing some dead API calls and adding a few tests for others.

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -13,68 +13,39 @@
...
-//     return $request->user();
-// });
-
-Route::get('/homepage_data', function () {
+Route::get('/homepage_data', function () { // Used from DeviceController, tested.
     return App\Http\Controllers\ApiController::homepage_data();
 });
 
-Route::get('/party/{id}/stats', function ($id) {
+Route::get('/party/{id}/stats', function ($id) { // Not used but worth keeping, tested.
```
Comment (ngm) [inline routes/api.php:20]: This is used from trp.org
(/var/www/therestartproject.test/wp-content/themes/restart/js/load-specific/fixometer-stats.js)

File: routes/api.php [hunk 2]
```diff
@@ -13,68 +13,39 @@
...
     return App\Http\Controllers\ApiController::partyStats($id);
 });
 
-Route::get('/group/{id}/stats', function ($id) {
+Route::get('/group/{id}/stats', function ($id) { // Not used but worth keeping, tested.
```
Comment (ngm) [inline routes/api.php:24]: This is used from trp.org
(/var/www/therestartproject.test/wp-content/themes/restart/js/load-specific/fixometer-stats.js)

**Review by ngm — APPROVED** [review]
Great.  All looks good to me.  

Lovely to see so much old code being removed...

File: routes/api.php (hunk 1 above)
Comment (edwh) [reply routes/api.php]: Ok - added a comment.

File: routes/api.php (hunk 2 above)
Comment (edwh) [reply routes/api.php]: Added another comment.

**Pull request merged**